Updated bootstrap and bootswatch

ceiphr opened the pull request:
Updated bootstrap and bootswatch. `Yarn.lock` was remade due to some issues adding bootstrap and bootswatch version 4.3.1 to yarn.

devinmatte commented:
@ceiphr If you need help fixing your yarn file reach out, it shouldn't need to be remade like that

ceiphr commented:
I've updated `yarn.lock` using node version v8.9.1 instead of v10 to resolve the issues I was facing with yarn. I also tweaked the `package.json` and removed the package-lock file like you asked.
